src/patternfly/components/Table/table.scss

```diff
@@ -432,26 +432,23 @@
 // Compact table
 // ==================================================================
 .pf-c-table.pf-m-compact {
+  &:not(.pf-m-border-rows) {
+    tbody {
+      --pf-c-table--BorderWidth: 0;
+      --pf-c-table--BorderColor: transparent;
+
+      tr:first-child > * {
+        --pf-c-table-cell--PaddingTop: calc(var(--pf-c-table--m-compact-cell--PaddingTop) * 2);
```

Comment:
What was the reason for this change? Its now making the compact table in the compound expansion look off:

<img width="953" alt="Screen Shot 2019-03-14 at 9 13 10 AM" src="https://user-images.githubusercontent.com/20118816/54359661-829e3f80-4639-11e9-9a11-3d5ff31a1db7.png">